homework5/ParsingTree/ParsingTree/NumberNode.cs

```diff
@@ -0,0 +1,36 @@
+namespace ParsingTree;
+
+/// <summary>
+/// The node with integer value.
+/// </summary>
+public class NumberNode : Node
+{
+      private readonly int value;
+
+      /// <summary>
+      /// Initializes a new instance of the <see cref="NumberNode"/> class.
+      /// </summary>
+      /// <param name="value">Node`s value.</param>
+      public NumberNode(int value)
+      {
+            this.value = value;
+      }
+
+      /// <summary>
+      /// The function for calculating the value.
+      /// </summary>
+      /// <returns>Node`s value.</returns>
+      public override int Evaluate()
+      {
+            return this.value;
+      }
```

Comment:
```suggestion
      public override int Evaluate() => this.value;
```
Если в методе (конструкторе) только строчка return, всегда можно компактнее записать